tts-service/service/service.go

```diff
@@ -95,6 +98,30 @@ func (srv impl) Get(id string) (*TtsResult, error) {
 	}, nil
 }
 
+func (srv impl) Delete(id string) error {
+
+	// Get tts media id
+	ttsResult, getErr := srv.Get(id)
+	if getErr != nil {
+		return getErr
+	}
+	mediaId := ttsResult.MediaId
+
+	// Remove metadata from persistance
+	persistanceErr := srv.persistence.del(id)
+	if persistanceErr != nil {
```

Comment:
I think we should continue with media removal even if metadata returns an error.
Good advice here: For such a multi-step operation where there's no transactional context, always try to implement it with "idempotence" in mind: that is the user can safely try to repeat entire operation.
This is especially important when dealing with REST-based API's since network failures are inevitable, and we want to have simple way of recovering from them (repeating failed operation is simplest option).